src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration.jsx

```diff
@@ -66,33 +94,64 @@ const defaultPlotStylingConfig = {
       },
     },
   ],
-  pointSize: 5,
-  pointOpa: 5,
-  pointStyle: 'circle',
-  toggleInvert: '#FFFFFF',
 };
 
-// This will be taken with a useSelector eventually
-const persistedConfigs = {
-  samplePlot: _.cloneDeep(defaultPlotStylingConfig),
-  frequencyPlot: _.cloneDeep(defaultPlotStylingConfig),
-  elbowPlot: _.cloneDeep(defaultPlotStylingConfig),
+const frequencyPlotConfigRedux = {
+  uuid: 'dataIntegrationFrequency',
+  type: 'dataIntegrationFrequency',
 };
 
 const DataIntegration = () => {
   const { Panel } = Collapse;
+
+  const dispatch = useDispatch();
   const router = useRouter();
   const { experimentId } = router.query;
 
-  const [activePlotKey, setActivePlotKey] = useState('elbowPlot');
-  const [config, setCurrentConfig] = useState(persistedConfigs.elbowPlot);
+  // This will be taken with a useSelector eventually
+  const persistedConfigs = {
+    samplePlot: _.cloneDeep(defaultElbowPlotStylingConfig),
+    frequencyPlot: useSelector((state) => state.componentConfig[frequencyPlotConfigRedux.uuid]?.config),
+    elbowPlot: _.cloneDeep(defaultElbowPlotStylingConfig),
+  };
+
+  const [activePlotKey, setActivePlotKey] = useState('frequencyPlot');
+  const config = persistedConfigs[activePlotKey];
+
+  const setCurrentConfig = () => {
+    // This will be used for dispatching the config updates to redux
+  };
```

Comment:
if it won't be used for these changes, I suggest that you remove it -- we can add it during the work on redux